README.md

```diff
@@ -147,6 +147,58 @@ If the instance is a `DataModel`, this method will throw. Places should be saved
 
 Throws on error.
 
+### `remodel.writeNewPlaceAsset`
+```
+remodel.writeNewPlaceAsset(instance: DataModel, options: Options): string
+
+where Options: {
+	name: string,
+	description: string?,
+	isPublic: boolean?,
+	allowComments: boolean?,
+}
+```
+
+Uploads the given `DataModel` instance to Roblox.com as a new place with the corresponding options.
+
+The options: `description`, `isPublic`, and `allowComments` are all optional.
+``description`` default to an empty string. ``isPublic`` and ``allowComments`` default to ``false``.
```

Comment:
Having a single newline here doesn't really do anything in markdown, so we should merge these lines.

Additionally, can you change the double backticks to single backticks? I don't think I understand why they work here.

Reply:
Yeah, I'm not sure why I did the double backticks. I think I was thinking of discord formatting.

I'm assuming you meant to combine the options lines.